demo/list-and-download/iterable/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+# GKE deployment script for the Dataflux iterable-style demo loop.
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: mirvine-dataflux
```

Comment:
Is it ok for this to be same for all runs of the demo or will it cause any conflicts in rare cases where someone attempts to run this demo when a previous run is in progress?

Reply:
You need to delete any existing runs with the same name, even completed, before you can kick off a new one. I want to look into it a bit more next, but I don't see any really convenient way to pass things like the job name or flags as parameters with Kubernetes unfortunately.